ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/IllegalSignatureCreationTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.jsonrpcproxy;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import tech.pegasys.ethsigner.core.jsonrpc.EthSendTransactionJsonParameters;
+import tech.pegasys.ethsigner.core.requesthandler.sendtransaction.transaction.EthTransaction;
+import tech.pegasys.ethsigner.core.signing.Signature;
+import tech.pegasys.ethsigner.signer.filebased.CredentialTransactionSigner;
+
+import org.junit.jupiter.api.Test;
+import org.web3j.crypto.Credentials;
+
+class IllegalSignatureCreationTest {
+
+  @Test
+  void ensureZenDesk32060IsResolved() {
```

Comment:
Rather a better name for this, without going to zen desk I have no idea what this test case is for. Maybe something along the lines of signingDataCreatesSignatureWithPositiveValues ?

Reply:
We have a public Github issue for this, see #247